### DISCUSSION TITLE
mentoring

### DISCUSSION THREAD

**Discussion by shaitanaleksander** [ratifire/devrate-service#42] [Ideas]
a person can be mentored by the interviewer, the idea is that when a person passes the interview, he can get feedback from the interviewer and after some time (a month for ex) he can pass the interview again, to see if his skill improved

**Comment by ghost** [comment]
Також, можна передбачити функцію виставлення і респондентом оцінки за менторство, яка буде відображатися в статистиці та рейтингу дева. Можливо буде цікавим, як девам які хотіли б претендувати на місце тімліда в якійсь компанії, так і айчарам, які будуть шукати людину з досвідом менторства.
З юридичної т.з., ніби проблем не повинно бути у зв'язку з безкоштовними консультаціями.
